index.ts

```diff
@@ -67,17 +67,21 @@ const loadURL = serve({ directory: `${__dirname}/dist` });
 	const handleSetup = () =>
 		ipcMain.handle(
 			"setup",
-			async function (event, { endpoint, bucket, accessKey, secretKey }) {
+			async function (
+				event,
+				{ endpoint, bucket, accessKey, secretKey, resticPassword }
+			) {
 				console.log("setup()", ...arguments);
 
 				const credentials = {
 					endpoint,
 					bucket,
 					accessKey,
-					secretKey
+					secretKey,
+					resticPassword
 				};
 
-				const resticPassphrase = "a";
+				const resticPassphrase = resticPassword || "a";
```

Comment:
Remove the fallback and add a sanity check here as well

Reply:
Please expand on this. Remove `const resticPassphrase = resticPassword || "a";`? What does `add a sanity check` mean?